lib/id3c/cli/command/geocode.py

```diff
@@ -320,6 +326,7 @@ def geocode_address(address: dict) -> dict:
     if not CLIENT:
         CLIENT = smartystreets_client_builder().build_us_street_api_client()
 
+    print('setup client')
```

Comment:
This gets printed to the output file when using `--log-ouput`

Reply:
Oops! Thanks for catching that